requirements.txt

```diff
@@ -2,15 +2,13 @@ accelerate==1.5.2
 datasets==3.4.1
 gguf==0.14.0
 huggingface-hub==0.29.3
+joblib==1.4.2
 langchain-community==0.3.18
 langchain-openai==0.3.6
 llama-cpp-python==0.3.8
 matplotlib==3.10.0
 numpy==1.26.4
 openai==1.63.2
-pandas==2.2.1
-scikit-learn==1.6.1
```

Comment:
I think scikit-learn and pandas are still required for the traditional approach. Please keep these requirements.